packages/nocms-plugin-image-resources/test/createImageResourceProvider.spec.js

```diff
@@ -0,0 +1,94 @@
+import assert from 'assert';
```

Comment:
I still advice to stick with chai.expect, as we mostly use that even though `assert` is used in other plugins - they could be converted later.